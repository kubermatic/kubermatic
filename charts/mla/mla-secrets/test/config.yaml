../../../config/mla-secrets/values.yaml