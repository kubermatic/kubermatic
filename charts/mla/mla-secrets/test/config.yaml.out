---
# Source: mla-secrets/templates/grafana-secret.yaml
apiVersion: v1
kind: Secret
metadata:
  name: grafana
type: Opaque
data:
  admin-user: "YWRtaW4="
  admin-password: "OVBUSnVnTGxKWnl6RVE2MFFnQzdsMmV6NzZqM1I4WFBPeFBkSVN0Wg=="
---
# Source: mla-secrets/templates/minio-secret.yaml
apiVersion: v1
kind: Secret
metadata:
  name: minio
type: Opaque
data:
  rootUser: "ckV1SkcyNTYxTGRnT253SXgydmM="
  rootPassword: "Rkx4TVNmZWh5T0ZTWWlJWVpPdzR3QWNVSnJaVEZXZ3pCdm8yUVpDSQ=="
